tripal_jbrowse_instance.module

```diff
@@ -81,6 +81,48 @@ function tripal_jbrowse_instance_menu() {
   return $items;
 }
 
+function tripal_jbrowse_block_info() {
+  $blocks['tripal_jbrowse_list'] = [
+    'info' => 'Tripal JBrowse List',
+  ];
+
+  return $blocks;
+}
+
+function tripal_jbrowse_block_view($delta = '') {
+  $block = [];
+
+  switch ($delta) {
+    case 'tripal_jbrowse_list':
+      $records = db_query('
+        SELECT * 
+        FROM {tripal_jbrowse_instance} ti 
+        JOIN {chado.organism} o 
+        ON ti.organism_id=o.organism_id 
+        ORDER BY o.genus, o.species');
+      $header = ['Organism Name', 'Link to JBrowse'];
+      $rows = [];
+      foreach ($records as $record) {
+        tripal_jbrowse_instance_parse_link($record);
+        $entity_id = chado_get_record_entity_by_table('organism', $record->organism_id);
+        // $organism = chado_get_organism(['organism_id' => $record->organism_id]);
+        $name = $record->genus . ' ' . $record->species;
+        $rows[] = [
+          l($name, '/bio_data/' . $entity_id),
```

Comment:
Adding `?tripal_pane=group_jbrowse` will open the jbrowse pane for the organism which I think is appropriate here.
```suggestion
          l($name, '/bio_data/' . $entity_id . '?tripal_pane=group_jbrowse'),
```